router/view/templates/styles/input.css

```
@@ -37,3 +37,22 @@ select {
 /* Channel item selected state */
 .channel-item.selected { @apply bg-zinc-100; }
 .dark .channel-item.selected { @apply bg-zinc-700; }
+
+/* Loading states */
+.loading-spinner {
+  border: 2px solid currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)